### ISSUE TITLE
tool not working

### ISSUE THREAD

**Issue by Wolfy530** [zoony1337/TikTok-Shares-Botter#9]
i think they patched it or your tool isn't working anymore. it was working in the morning

**Comment by epicmines33** [comment]
yeah i think tiktok patched it as it was working fine literally a few hours ago for me

**Comment by Wolfy530** [comment]
> yeah i think tiktok patched it as it was working fine literally a few hours ago for me

yep, i tried my friend tool and its not working

**Comment by epicmines33** [comment]
> > yeah i think tiktok patched it as it was working fine literally a few hours ago for me
> 
> yep, i tried my friend tool and its not working

yeah i think tiktok did a patch for share bots in general cause its become a more popular thing recently

**Comment by ClipsyEU** [comment]
it has been patched and it will be patched for ever now you need a tiktok account in order to share a video rip

**Comment by GeniusDev0** [comment]
probably not gonna work. tiktok made it so you need an account to do this. maybe we also enter an alt account's details in the future and the alt account will bot it.
